source/guide_eqdkp-plus.rst

```diff
@@ -0,0 +1,83 @@
+.. highlight:: console
+.. author:: GodMod <godmod@eqdkp-plus.eu>
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/eqdkp-plus.svg
+      :align: center
+
+#########
+EQdkp Plus
+#########
+
+EQdkp-Plus_ is an open source Content Management System (CMS) and Guild Management System in PHP and distributed unter the AGPLv3 licence.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * PHP_
+  * MySQL_
+  * domains_
+
+Prerequisites
+=============
+
+We're using PHP_ in the stable version 7.1:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '7.1'
+ [isabell@stardust ~]$
+
+.. include:: includes/my-print-defaults.rst
+
+If you want to use your EQdkp Plus with your own domain you need to setup your domain first:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+``cd`` to your `document root`_, then download the latest release of EQdkp Plus and extract it:
+
+::
+
+ [isabell@stardust ~]$ cd ~/html/
+ [isabell@stardust html]$ curl -L https://eqdkp-plus.eu/repository/download/latestcore -o eqdkp-plus.zip
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+  100 25565    0 25565    0     0  22942      0 --:--:--  0:00:01 --:--:-- 22948
+  100 5138k  100 5138k    0     0   769k      0  0:00:06  0:00:06 --:--:-- 1108k
+ [isabell@stardust html]$ unzip eqdkp-plus.zip
+
+Now point your browser to your uberspace URL and follow the instructions of the Installer Assistent.
+
```

Comment:
Can you elaborate on the FTP Settings?

Reply:
The newest version of EQdkp Plus does not show any FTP settings anymore, only if there are problems (which ubespace not has). The download link for the latest version pointed to the wrong version, therefore you installed an older version which still has FTP settings.